docusaurus/docs/iOS/swiftui/components/channel-list-header.md

```diff
@@ -27,6 +27,14 @@ let appearance = Appearance(colors: colors)
 let streamChat = StreamChat(chatClient: chatClient, appearance: appearance)
 ```
 
+By default, the SDK shows a collapsed navigation bar. If you want to use a large navigation bar that's collapsible on scrolling down, you can configure it via the `navigationBarDisplayMode` method in the `ViewFactory`.
+
+```swift
+func navigationBarDisplayMode() -> NavigationBarItem.TitleDisplayMode {
+    .large
+}
+```
+
```

Comment:
Might also be an idea to have that in a config somewhere. Are there downsides to that? A config enum could be a little easier than a factory method to implement...

Reply:
As discussed, we will keep it until we have a need for channel list config.